src/searchengine/nova3/novaprinter.py

```diff
@@ -24,6 +24,8 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+from __future__ import print_function
```

Comment:
This is pointless since this file is python3 **ONLY**.

Reply:
The point is that duplicate code is both useless and dangerous.

Reply:
Thanks for the concern, we plan dropping all python2 code when macOS stop shipping python2 by default.